flutter/news/test/core/network/network_info_test.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:internet_connection_checker/internet_connection_checker.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:news/core/network/network_info.dart';
+
+import 'network_info_test.mocks.dart';
+
+@GenerateMocks([InternetConnectionChecker])
+void main() {
+  late NetworkInfoImpl networkInfoImpl;
+  late MockInternetConnectionChecker mockInternetConnectionChecker;
+
+  setUp(() {
+    mockInternetConnectionChecker = MockInternetConnectionChecker();
+    networkInfoImpl = NetworkInfoImpl(mockInternetConnectionChecker);
+  });
+
+  group("isConnected", () {
+    test(
+      'should forward the call to InternetConnectChecker.hasConnection',
+      () async {
+        // arrange
```

Comment:
You don't need the `//arrange`, etc, comments

Just group each of the 3 parts. Like this

```
final tHasConnectionFuture = Future.value(true);
when(mockInternetConnectionChecker.hasConnection)
         .thenAnswer((_) => tHasConnectionFuture);

final result = networkInfoImpl.isConnected;

verify(mockInternetConnectionChecker.hasConnection);
expect(result, tHasConnectionFuture);
```

Reply:
updated on https://github.com/novoda/multiplatform-playground/commit/8323a52c3c021a85d1662fa2609c061534a5fce3